client/components/status/status.test.js

```diff
@@ -1990,3 +2015,51 @@ describe("<Status /> interactions", () => {
     });
   });
 });
+
+describe("getUserRadiusUsage method", () => {
```

Comment:
Doesn't this replicate the same tests that are performed by `test getUserRadiusUsage method` ?

What we need here is a snapshot test with the following scenarios: 

1. `checks` is not present in the response 
2. `checks` is set to `[]` in the response 

In both  the cases, the following section should not present in the output. 

https://github.com/openwisp/openwisp-wifi-login-pages/blob/184ae2a0f01c514ff5aef3ccbc32fc36817f61ce/client/components/status/status.js#L1039-L1093

Reply:
You can either update this test or use it as reference for a new test 

https://github.com/openwisp/openwisp-wifi-login-pages/blob/4911b491af399537419b3c786e6c47399fc53e2a/client/components/status/status.test.js#L1826-L1872